src/components/Popover/components/Trigger/Trigger.tsx

```diff
@@ -81,7 +83,7 @@ export const Trigger = ({
         <div
```

Comment:
This div is not accessible from the keyboard, we need to add `tabIndex={0}` and focus styles

Reply:
I think it is applicable only for cases when Trigger wraps non-interactive elements, not accessible from keyboard by default. Otherwise, if `<button>` is the child node, the Trigger itself will be focused first, then `<button>` element. Seems there is unnecessary focusable element.

Maybe one should pass `tabIndex` from props, at the developer's discretion?

Reply:
The problem is we have no control in trigger node. We will lose a11y if user won't provide proper element to it

Reply:
Accessible element should be inside the trigger element, but the trigger itself must not be, to avoid focus duplication